chapters/history.md

```diff
@@ -109,4 +109,4 @@ This chapter prepared you to...
 @teacher3f
 @teacher4
 @teacher4a
-@teacher4d
\ No newline at end of file
+@teacher4d
```

Comment:
Not sure why this line showed up in the changelog for this pr when 4d already shows up in the live version of this chapter... ...